cadcutils/dev_requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy==1.12.0b1
+astropy==1.2.1
+cadcutils==1.0
```

Comment:
cadcutils==1.0 not required as the following line installs the local version.

Please remove it from vofs/dev_requirements.txt too - thanks